### ISSUE TITLE
Drakengard 3 [NPEB01407] - Random Freezes

### ISSUE THREAD

**Issue by daniel-fs** [RPCS3/rpcs3#8216]
Hello!

Roughly 2 months ago I posted an issue #7806 where the game Nier was randomly freezing. I'm linking that issue since it might be related to this one (might be an Intel driver problem).

Recently I started playing Drakengard 3 and I'm also getting random freezes while playing the game. However, I don't get the same errors as I did with Nier, and that is why I opened a new issue. When I close RPCS3, I get a warning message saying that it took too long to close and that a thread might have deadlocked.

In the RPCS3 interface logs there aren't any errors, last messages are usually normal stuff like `{RSX Decompiler Thread} RSX: New program compiled successfully`, but the attached log might have more useful info.

Like Nier, the freezes are unpredictable. I can get get them either every 10 minutes of gameplay or after 1-2 hours.

Let me know if any other info is needed.

## Details
RPCS3 v0.0.10-10370-cc723ed4

OS: Windows 10 64-bit
CPU: Intel Core i5-9300h
GPU: Intel UHD 630
RAM: 16 GB

GPU driver version: 26.20.100.7985

[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/4625412/RPCS3.zip)

**Comment by Andryshik345** [comment]
Try to disable `SPU ⁪Lower ⁪Thread ⁪Priority` and disable this all, it's not needed.
![изображение](https://user-images.githubusercontent.com/24612088/82026900-d4fb4600-96ac-11ea-9d69-7888954d6cc4.png)

**Comment by daniel-fs** [comment]
> Try to disable `SPU ⁪Lower ⁪Thread ⁪Priority` and disable this all, it's not needed.
> ![изображение](https://user-images.githubusercontent.com/24612088/82026900-d4fb4600-96ac-11ea-9d69-7888954d6cc4.png)

I didn't even remember I had these on, it was from back when I was playing around with the settings for Nier...

Anyway, it made no difference, still got a freeze/deadlock after playing for a while. Attached the log, now with the most recent version of RPCS3 at the time of writing this.

RPCS3 v0.0.10-10383-b5dbb2eb

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/4633758/RPCS3.log)

**Comment by daniel-fs** [comment]
Little update...

So, the deadlocks were being pretty bad on Chapter 2 Verse 5. Chapter 2 as a whole seems very prone to the freezes, but Verse 5 in particular I literally couldn't get to the end of it.

After playing around with the settings, clearing caches, etc, I decided to try the OpenGL renderer.

And the freezes seem to be gone... the framerate drops are atrocious when the shaders are compiling, but after that it runs as well as Vulkan. I will keep playing with OpenGL and see if I get any freeze and report here, but for now I'd say this is either a Vulkan related problem or the Intel GPU driver being faulty with Vulkan.

**Comment by Andryshik345** [comment]
Nope, all good for me on discrete GPU(GTX 750 Ti) with Vulkan, so then it's related to iGPU.

**Comment by AniLeo** [comment]
Since other people are not getting such issue on dedicated GPUs, it's safe to assume it's likely an iGPU issue unless proven otherwise
You should try this on a Linux distro with up-to-date Mesa 20.2 / 20.3-dev drivers and see if it still happens

**Comment by C0LT0N13** [comment]
I am getting the same error, and I have an RTX 2060.

**Comment by VL4DST3R** [comment]
Same issue here while playing `Hatsune Miku: Project DIVA F2nd` , never happened before, running on a 3070.
@AniLeo no Intel/iGPU at all in my case since i'm running on an 5900X.

**Comment by Megamouse** [comment]
This should be tested again without patches on the latest build.
There was a 60fps patch applied.
Issues created with patches active are invalid.
